src/Tribe/Rewrite.php

```diff
@@ -247,15 +247,15 @@ public function get_bases( $method = 'regex' ) {
 		 * @var array $bases
 		 */
 		$bases = apply_filters( 'tribe_events_rewrite_base_slugs', array(
-			'month' => array( 'month', $tec->monthSlug ),
-			'list' => array( 'list', $tec->listSlug ),
-			'today' => array( 'today', $tec->todaySlug ),
-			'day' => array( 'day', $tec->daySlug ),
-			'tag' => array( 'tag', $tec->tag_slug ),
-			'tax' => array( 'category', $tec->category_slug ),
-			'page' => (array) 'page',
-			'single' => (array) Tribe__Settings_Manager::get_option( 'singleEventSlug', 'event' ),
-			'archive' => (array) Tribe__Settings_Manager::get_option( 'eventsSlug', 'events' ),
+			'month'    => array( 'month', $tec->monthSlug ),
+			'list'     => array( 'list', $tec->listSlug ),
+			'today'    => array( 'today', $tec->todaySlug ),
+			'day'      => array( 'day', $tec->daySlug ),
+			'tag'      => array( 'tag', $tec->tag_slug ),
+			'tax'      => array( 'category', $tec->category_slug ),
+			'page'     => (array) esc_html_x( 'page', 'The "/page/" URL string component.', 'the-events-calendar' ),
```

Comment:
We need an array of the translated and the Englis version, just like the ones above. So we don't loose the Canonical english version.